packages/template-cli/templates/src/ssr/package.json

```diff
@@ -1,5 +1,6 @@
 {
-  "name": "webapp",
+  "name": "PROJECT_NAME",
```

Comment:
@dnitsch are these going to be replaced from the CLI?

Reply:
Yes. Just as the previous comment